webapp/templates/footer.html.tmpl

```diff
@@ -5,4 +5,4 @@
     <li class="footer-list-item">{{ .copyright }}</li>
   </ul>
 </footer>
-{{ end }}
```

Comment:
NIT (not needing to be addressed): kinda prefer line breaks at the end do we have a policy around this?

Reply:
I guess `.tmpl` files are not covered by the style formatter, but yes, linefeed should always be ensured. Maybe we should add an [`.editorconfig`](https://editorconfig.org/) file.